assets/stylesheets/layout/_main.scss

```diff
@@ -372,15 +381,15 @@ $sidebar-width-min: 228px;
 .wp-secondary .site-selector {
 	background: lighten( $gray, 30% );
 	border-right: 1px solid lighten( $gray, 25% );
-	position: fixed;
-		top: 47px;
+	position: absolute;
```

Comment:
It'd be good for @shaunandrews to give this a quick look to make sure there are no regressions since he spent a lot of time getting this to behave properly.